README.md

```diff
@@ -1,70 +1,39 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white) ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white) ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
 
-## Available Scripts
 
-In the project directory, you can run:
+# Math Magicians 🧮
+> React Web App for math lovers.
```

Comment:
- [ **OPTIONAL** ] What do you think if you improve your project's description with a short resume about the project to make it more professional? I can suggest you gest inspiration from the one form **[project's requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md#:~:text=%22Math%20magicians%22%20is,to%20use%20React.)**